zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/elasticsearch/ElasticsearchEngineClient.java

```diff
@@ -227,6 +227,29 @@ public boolean importersCompleted(
     }
   }
 
+  /**
+   * Overwrites the settings of the existing index template with the settings block in the
+   * descriptor schema json file.
+   *
+   * @param indexTemplateDescriptor of the index template to have its settings overwritten.
+   */
+  @Override
+  public void updateIndexTemplateSettings(
+      final IndexTemplateDescriptor indexTemplateDescriptor, final IndexSettings currentSettings) {
+    final var updateIndexTemplateSettingsRequest =
+        updateTemplateSettings(indexTemplateDescriptor, currentSettings);
+
+    try {
+      client.indices().putIndexTemplate(updateIndexTemplateSettingsRequest);
+    } catch (final IOException | ElasticsearchException e) {
+      throw new ElasticsearchExporterException(
+          String.format(
+              "Failed to update index template settings for [%s]",
+              indexTemplateDescriptor.getTemplateName()),
+          e);
```

Comment:
🔧 Let's try to follow our error guidelines https://github.com/camunda/camunda/wiki/Error-Guidelines and give some more context

Example:

```suggestion
      throw new ElasticsearchExporterException(
          String.format(
              "Expected to update index template settings '%s' with '%s', but failed ",
              indexTemplateDescriptor.getTemplateName(),
              updateIndexTemplateSettingsRequest),
          e);
```